src/model/ModelHVAC.i

```diff
@@ -176,6 +176,7 @@ MODELOBJECT_TEMPLATES(CoilCoolingWaterToAirHeatPumpVariableSpeedEquationFitSpeed
 MODELOBJECT_TEMPLATES(CoilHeatingDXMultiSpeedStageData);
 MODELOBJECT_TEMPLATES(CoilHeatingDXVariableRefrigerantFlow);
 MODELOBJECT_TEMPLATES(CoilHeatingGasMultiStageStageData);
+MODELOBJECT_TEMPLATES(CoilHeatingElectricMultiStageStageData);
```

Comment:
Definitely missing an %ignore directive for C# partial classes here

Reply:
Then it was missing for `CoilHeatingGasMultiStage`...

Reply:
perfectly possible indeed :)